test/new-e2e/runner/parameters/fixtures/config_with_stackparams.yaml

```diff
@@ -1,12 +1,14 @@
 # Environments configuration, validated and passed to Pulumi
 configParams:
   aws:
+    # aws account where resources will be created on
+    account: "kiki"
```

Comment:
Is this name correct?

Reply:
This is used in a test, might rename `fixture` to `testfixture` to be more explicit. The logic here is different from the one in `test-infra`, as I don't check for the validity of the account. The validation is at `inv setup` level in `test-infra`, once here I expect the local config to be valid, so I accept any value.